.github/workflows/release.yml

```diff
@@ -103,6 +105,7 @@ jobs:
         if: ${{ startsWith(github.ref, 'refs/heads/release') }}
         env:
           AIRDROP_WEBSOCKET_CRYPTO_KEY: ${{ secrets.PROD_AIRDROP_WEBSOCKET_CRYPTO_KEY }}
+          BINARY_NAME: ${{ runner.os == 'Linux' && 'tari_universe' || 'Tari Universe' }}
```

Comment:
Add a comment above, what/why it's been done. This is something I think will be forgetten and removed and added back in.